Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -34,9 +36,16 @@ ErrorOr<void> PresenterWidget::initialize_menubar()
             return;
         this->set_file(response.value()->filename());
     });
+
+    m_settings_window = TRY(GUI::SettingsWindow::create("Presenter Settings", GUI::SettingsWindow::ShowDefaultsButton::No, GUI::SettingsWindow::CloseProgramAfterCompletion::No));
+    (void)TRY(m_settings_window->add_tab<PresenterSettingsFooterWidget>("Footer", "footer"sv));
+    auto settings_action = GUI::Action::create("&Settings...", [this](auto&) {
+        m_settings_window->show();
+    });
```

Comment:
I'd suggest late-initializing the SettingsWindow in the action instead, since most of the time the user won't need to open it.

Reply:
Then I cannot use TRY for the initialization, so which is better?